packages/frontend/src/views/Modify.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="modify">
+    <div>
+      <div>
+        <span class="mr-2">{{ $t('choose_a_desk_or') }}</span>
+        <v-btn small color="primary" @click="click">{{ $t('any_desk') }}</v-btn>
+      </div>
+
+      <v-btn small color="primary" class="p-0 mt-5" @click="$router.go(-1)"><v-icon small>fa-times</v-icon></v-btn>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+import ActionBar from '@/components/ActionBar.vue';
+
+@Component
+export default class Modify extends Vue {
+  click() {
+    this.$router.push({ path: '/book-desk' });
```

Comment:
Please use the name of the route here.

```suggestion
    this.$router.push({ name: 'book-desk' });
```

Reply:
I used the name but some problem occurs that's the reason I used the path. I need suggestion in that case how to use this.